perun-core/src/main/java/cz/metacentrum/perun/core/entry/MembersManagerEntry.java

```diff
@@ -1292,8 +1294,8 @@ public void sendPasswordResetLinkEmail(PerunSession sess, Member member, String
 	public RichMember createSponsoredMember(PerunSession session, Vo vo, String namespace, Map<String, String> name, String password, User sponsor)
 			throws InternalErrorException, PrivilegeException, AlreadyMemberException,
 			LoginNotExistsException, PasswordCreationFailedException,
-		ExtendMembershipException,
-			WrongAttributeValueException, ExtSourceNotExistsException, WrongReferenceAttributeValueException, UserNotInRoleException {
+			ExtendMembershipException,
```

Comment:
Why is this exception alone on the line?

Reply:
I aligned all exceptions in three rows.